staff/almu-itur/cv-styled/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>CV ALMUDENA ITURGOYEN</title>
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css" />
+</head>
+<body>
+    <header>
+            <h1>ALMUDENA ITURGOYEN</h1>
+            <h2>Junior front-end developer</h2>
+            <p><strong>Email:</strong> almudena.iturgoyen@gmail.com</p>
+            <p><strong>Phone:</strong> 650239067</p>
```

Comment:
be aware that the info you put in github is public

Reply:
Changed! Thanks